deploy/manifests/prometheus/crd.yaml

```diff
@@ -0,0 +1,266 @@
+---
```

Comment:
Like @rainest commented above:

> We shouldn't include the PodMonitor CRD ourselves. That should come from the Prometheus operator. We should only provide the PodMonitor/ServiceMonitor resources alone.

Please delete the CRD from the kustomization.